### DISCUSSION TITLE
How to update a chroot that’s in a usb

### DISCUSSION THREAD

**Discussion by INFOTECHadmin** [dnschneid/crouton#4970] [Q&A]
How can I update a chroot that’s in a usb drive with crouton -u command is there a way for me to specify where the chroot I want to update is as it defaults to /usr/local/chroots but my chroot is located in /media/removable/(name-of-removeable-media) I can’t move the chroot back to /usr/local/chroots due to storage limitations and update it

**Comment by CroutonIsFun** [comment]
Go to /usr/local and remove the shortcut:

`sudo rm chroots`

Now create a shortcut to your USB:

`sudo ln -s /media/removable/whatever chroots`

Now do a normal update:

`sudo crouton -n xenial -u` or whatever your chroot is called.

**Reply by INFOTECHadmin** [reply]
I get a response saying 
Ln: failed to create symbolic link ‘chroots’: Read-only file system, I just did `sudo ln -s /media/removable/whatever /usr/local/chroots ` and that seemed to accomplish the same thing
